### PR TITLE
Document intented usage of account&spend -related API

### PR DESCRIPTION
**Pull request by lucafavatella**
I am moving here items out of the epoch release notes that constitute intended usage.

### CODE REVIEW

File: epoch_api/spend_api_usage.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+The reason is that in order to spend tokens from your account you need to obtain tokens in the first place, and in order to obtain tokens you need to mine a block faster than all other miners in the network.
+Testing spending tokens in a local network enables obtaining tokens without competing with other miners.
+
+At least one of the two nodes needs to mine - in order to have tokens to spend.
+
+At least one of the two nodes needs to be configured with the peer address of the other node in order for them to form a network.
+
+### Running multiple nodes on same host
+
+Unless you are using Docker, in order to run two nodes on the same host you need to change the node name of each node.
```
Comment (dadaphl) [inline epoch_api/spend_api_usage.md]: I assume, that means, you have to have at least two copies of the epoch software (source and/or build) on your machine in two (or more) different locations? If yes, does that mean you have to change the node name before or after building?

Comment (lucafavatella) [reply epoch_api/spend_api_usage.md]: After having deployed the release binary (that is already built) though before starting the node. Let me clarify the text.

File: epoch_api/spend_api_usage.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The simplest way for testing sending tokens is setting up a local network of two nodes - disconnected from the testnet.
+The reason is that in order to spend tokens from your account you need to obtain tokens in the first place, and in order to obtain tokens you need to mine a block faster than all other miners in the network.
+Testing spending tokens in a local network enables obtaining tokens without competing with other miners.
+
+At least one of the two nodes needs to mine - in order to have tokens to spend.
+
+At least one of the two nodes needs to be configured with the peer address of the other node in order for them to form a network.
+
+### Running multiple nodes on same host
```
Comment (michalzajda) [inline epoch_api/spend_api_usage.md]: I think its good to mention here that this is mainly purposed for development (and simulating multi node env in Erlang friendly way)

File: epoch_api/spend_api_usage.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Unless you are using Docker, in order to run two nodes on the same host you need to change the node name of each node.
+For each node, after having deployed it - and before starting it - you need to change directory (`cd`) to the directory where you deployed the node, then for node 1 run:
```
Comment (lucafavatella) [inline epoch_api/spend_api_usage.md]: and `node 1` shall not be here

Comment (lucafavatella) [reply epoch_api/spend_api_usage.md]: well at least needs rephrasing

File: epoch_api/spend_api_usage.md (hunk 2 above)
Comment (lucafavatella) [reply epoch_api/spend_api_usage.md]: Note added. Thanks.

File: epoch_api/spend_api_usage.md (hunk 1 above)
Comment (lucafavatella) [reply epoch_api/spend_api_usage.md]: Addressed.

**Comment by lucafavatella** [comment]
@dadaphl Could you please re-review?

**Comment by lucafavatella** [comment]
I see 3 reviewers accepted. Merging.

**Pull request merged**